cpp/ql/test/library-tests/dataflow/taint-tests/localTaint.expected

```diff
@@ -5113,10 +5113,11 @@
 | swap1.cpp:109:5:109:30 | ... = ... | swap1.cpp:111:20:111:24 | data1 |  |
 | swap1.cpp:109:5:109:30 | ... = ... | swap1.cpp:115:18:115:22 | data1 |  |
 | swap1.cpp:109:23:109:28 | call to source | swap1.cpp:109:5:109:30 | ... = ... |  |
-| swap1.cpp:113:31:113:39 | call to move | swap1.cpp:113:31:113:51 | call to Class |  |
+| swap1.cpp:113:31:113:39 | call to move | swap1.cpp:113:31:113:51 | call to Class | TAINT |
```

Comment:
If I read this change correctly, that means that we will no longer have dataflow from `std::move(move_from)` to `IntWrapper::Class move_to(std::move(move_from))`, but only taint-flow. Is that correct?